src/app/team/documents/proofs/Edit.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Component, createRef } from 'react';
+
+import Header from '../../../../components/header';
+import Button from '../../../../components/button';
+import Selector from '../../../../components/selector';
+import Input from '../../../../components/input';
+
+const DEFAULT_PROOFS = [
+  'None',
+  'Photo ID',
+  'Proof of address',
+  'Proof of income',
+  'Birth certificates',
+];
+
+class ProofsRequiredEdit extends Component {
+  constructor(props) {
+    super(props);
+
+    let proofs;
+
+    if (this.props.value) {
+      proofs = this.props.value.map(el => el.document);
+    }
+
+    this.state = {
+      proofs,
```

Comment:
It's [kind of an anti-pattern](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html) to copy state into props, and especially so only in the constructor (since prop changes are ignored).
However, I don't think it's that big of a problem here, so this is fine.

Reply:
Yeah, this doesn't use any kind of props change, that's why i added it to the constructor only